runtime/src/lib.rs

```diff
@@ -851,44 +920,103 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 impl cumulus_pallet_dmp_queue::Config for Runtime {
     type Event = Event;
     type XcmExecutor = XcmExecutor<XcmConfig>;
-    type ExecuteOverweightOrigin = frame_system::EnsureRoot<AccountId>;
+    type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+}
+
+parameter_types! {
+	pub const Period: u32 = 6 * HOURS;
+	pub const Offset: u32 = 0;
+	pub const MaxAuthorities: u32 = 100_000;
```

Comment:
how did you determine 100k max authorities?
in 'master' we have 100

Reply:
I picked it from here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.13/runtime/src/lib.rs#L535. However I couldn't find this on master branch.

Reply:
i thought it might be similar to `validator_count` from the chain_spec.rs in 'master' branch